vertx-mssql-client/src/main/java/io/vertx/mssqlclient/MSSQLPool.java

```diff
@@ -60,56 +58,25 @@ static MSSQLPool pool(Vertx vertx, MSSQLConnectOptions connectOptions, PoolOptio
   @Override
   MSSQLPool preparedQuery(String s, Handler<AsyncResult<RowSet<Row>>> handler);
```

Comment:
I think these one-shot operations should be removed also. I know these are intended as "convenience methods" but my initial impression of using this library was I found it very confusing there were so many different ways to do the same thing. IMO we should only provide the user 1 API path where possible. 
The only benefit of these one-shot commands is that it is slightly less typing:
```java
// One-shot way
pool.preparedQuery(SQL, ar -> this::processRows);

// Regular way
pool.createPreparedQuery().execute(ar -> this::processRows);
```

If we get rid of the one-shot methods, it will significantly reduce the API surface and make the overall API much quicker for users to grasp.

Reply:
please see step 5.